### DISCUSSION TITLE
Calling .rpc to invoke postgres function says could not find function in schema cache

### DISCUSSION THREAD

**Discussion by sumchans** [supabase/supabase#3054] [Questions] [answered]
I am calling a supabase function from flutter/dart I get this error. I have read about cache reloading and automatic cache reloading, but not sure how to get fix it in my case.
**flutter: Could not find the public.new_conversation(members, the_conversationId) function in the schema cache**

This is how I call the function 
```
final response = await supabase.rpc('new_conversation', params: {
      'the_conversationId': conversationId,
      'members': participants
    }).execute();
```

**Comment by steve-chavez** [comment, marked as answer]
@sumchans Can you share your function definition?

`the_conversationId` <- this one looks suspcicious, it could be that you're not using a double quoted identifier.

**Reply by sumchans** [reply]
Thanks steve, That was exactly the issue.
